layouts/partials/fragments/contact.html

```diff
@@ -19,9 +19,9 @@
           class="contact"
           method="POST"
           name="{{- .Params.form_name | default (printf "contact-form-%s" .Name) }}"
-          id="{{ printf "contact-form-%s" .Name }}"
+          id="{{- .Params.form_name | default (printf "contact-form-%s" .Name) }}"
           {{- if .Params.netlify -}}
-            {{- safeHTMLAttr (print " netlify-recaptcha netlify data-has-netlify=\"true\"") -}}
+            {{- safeHTMLAttr (print " data-has-netlify=\"true\" netlify") -}}
```

Comment:
Should the netlify be the last attribute for it to work? That's weird. Also doesn't this disable the netlify recaptcha?

Reply:
Doesn't need to be last, just happened to be that way after my tests I think.

Nope it still has netlify recaptcha. The `netlify-recaptcha` config on the form itself is only used for custom provided recaptcha in combination with netlify. We don't support that yet.